packages/webpack/lib/setEnv.js

```diff
@@ -0,0 +1,26 @@
+module.exports = setEnv
+
+/**
+ * Replaces constants in your source code with values from `process.env`
+ * using the `webpack.EnvironmentPlugin`.
+ *
+ * Special feature: Using `setEnv` multiple times results in a single
+ * EnvironmentPlugin instance configured to do all the replacements.
+ *
+ * @param {string[]|object} constants
+ * @return {Function}
+ */
+function setEnv (constants) {
+  const setter = context => prevConfig => {
+    context.setEnv = Object.assign({}, context.setEnv, constants)
```

Comment:
Would it work this way?

```js
createConfig([
  setEnv({ NODE_ENV: 'development' }),
  setEnv(['DEBUG'])
])
```

Reply:
It wouldn't I guess, do we need a multi-setEnv functionality or it can be simplified to a straight `EnvironmentPlugin` call?

Reply:
Not sure, in this case we’ll need to parse options ourselves.

@andywer What do you think?

Reply:
We should definitely support having multiple `setEnv()`. And having the ability to pass an array like `setEnv(['DEBUG'])` that is resolved to `setEnv({ DEBUG: process.env.DEBUG })` would also be great.

I hope I got your question right.

Reply:
And thanks for the PR, @vlad-zhukov! :)

Reply:
The question is should we try to merge arguments from different `setEnv` calls ourselves and call the `EnvironmentPlugin` only once?

Reply:
I'd say yes. Seems cleaner. Not sure if multiple plugin instances have side effects.

Reply:
Cleaner to who? ;-) That would significantly complicate the code. The only side effect I can imagine is performance.

Reply:
Yeah, if it works seemlessly with multiple plugin instances and changing it would make the code noticeably more complex, then nevermind.
No need to make our lives unnecessarily hard 😉